terraform/environments/example/locals.tf

```diff
@@ -40,5 +40,8 @@ locals {
   # environment specfic variables
   # example usage:  
   # example_data = local.application_data.accounts[local.environment].example_var
-  application_data = fileexists("./application_variables.json") ? jsondecode(file("./application_variables.json")) : {}
+  # app_variables = fileexists("./application_variables.json") ? jsondecode(file("./application_variables.json")) : {}
```

Comment:
Why did this need to be changed from the template?  Is there an issue with the template?